t/test01.t

```diff
@@ -136,15 +137,25 @@ subtest 'add a first test' => sub {
     is( $hash_id_db, $hash_id , 'Correct hash_id in database' );
 
     # test test_progress API
-    is( $backend->test_progress( { test_id => $hash_id } ), 0 , 'API test_progress -> OK');
+    $progress = $backend->test_progress( { test_id => $hash_id } );
```

Comment:
Maybe declare the variable anew in each subtest where it's used? The value is not supposed to survive to the next subtest, so why should the variable?

Reply:
indeed there is no reason to declare `progress` only once, I applied your suggestion.